<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="description" content="Clue.io is a database of pertubations provided by the Broad institute.
    This package provides a programmatic interface to query the database from
    within R. Utility functions to query directly using differential gene expression
    results from DESeq2 are also included.">
<title>Query the Clue Database of Perturbations Using R â€¢ cluequery</title>
<script src="deps/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<link href="deps/bootstrap-5.1.0/bootstrap.min.css" rel="stylesheet">
<script src="deps/bootstrap-5.1.0/bootstrap.bundle.min.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- bootstrap-toc --><script src="https://cdn.rawgit.com/afeld/bootstrap-toc/v1.0.1/dist/bootstrap-toc.min.js"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- search --><script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/6.4.6/fuse.js" integrity="sha512-zv6Ywkjyktsohkbp9bb45V6tEMoWhzFzXis+LrMehmJZZSys19Yxf1dopHx7WzIKxr5tK2dVcYmaCk2uqdjF4A==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/autocomplete.js/0.38.0/autocomplete.jquery.min.js" integrity="sha512-GU9ayf+66Xx2TmpxqJpliWbT5PiGYxpaG8rfnBEk1LL8l1KGkRShhngwdXK1UgqhAzWpZHSiYPc09/NwDQIGyg==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js" integrity="sha512-5CYOlHXGh6QpOFA/TeTylKLWfB3ftPsde7AnmhuitiTX4K5SqCLBeKro6sPS8ilsz1Q4NRx3v8Ko2IBiszzdww==" crossorigin="anonymous"></script><!-- pkgdown --><script src="pkgdown.js"></script><meta property="og:title" content="Query the Clue Database of Perturbations Using R">
<meta property="og:description" content="Clue.io is a database of pertubations provided by the Broad institute.
    This package provides a programmatic interface to query the database from
    within R. Utility functions to query directly using differential gene expression
    results from DESeq2 are also included.">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body>
    <a href="#main" class="visually-hidden-focusable">Skip to contents</a>
    

    <nav class="navbar fixed-top navbar-light navbar-expand-lg bg-light"><div class="container">
    
    <a class="navbar-brand me-2" href="index.html">cluequery</a>

    <small class="nav-text text-muted me-auto" data-bs-toggle="tooltip" data-bs-placement="bottom" title="">0.3.3</small>

    
    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbar" aria-controls="navbar" aria-expanded="false" aria-label="Toggle navigation">
      <span class="navbar-toggler-icon"></span>
    </button>

    <div id="navbar" class="collapse navbar-collapse ms-2">
      <ul class="navbar-nav me-auto">
<li class="nav-item">
  <a class="nav-link" href="reference/index.html">Reference</a>
</li>
      </ul>
<form class="form-inline my-2 my-lg-0" role="search">
        <input type="search" class="form-control me-sm-2" aria-label="Toggle navigation" name="search-input" data-search-index="search.json" id="search-input" placeholder="Search for" autocomplete="off">
</form>

      <ul class="navbar-nav"></ul>
</div>

    
  </div>
</nav><div class="container template-home">
<div class="row">
  <main id="main" class="col-md-9"><div class="section level1">
<div class="page-header"><h1 id="cluer">clueR<a class="anchor" aria-label="anchor" href="#cluer"></a>
</h1></div>
<!-- badges: start -->
<!-- badges: end -->
<p>The goal of clueR is to provide an easy R interface to query Connectivity Map data from <a href="https://clue.io" class="external-link">Clue</a>.</p>
<div class="section level2">
<h2 id="installation">Installation<a class="anchor" aria-label="anchor" href="#installation"></a>
</h2>
<p>You can install the current version of clueR from <a href="https://github.com/labsyspharm/clueR" class="external-link">Github</a> with:</p>
<div class="sourceCode" id="cb1"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="kw">if</span> <span class="op">(</span><span class="op">!</span><span class="fu"><a href="https://rdrr.io/r/base/ns-load.html" class="external-link">requireNamespace</a></span><span class="op">(</span><span class="st">"remotes"</span>, quietly <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span><span class="op">)</span>
  <span class="fu"><a href="https://rdrr.io/r/utils/install.packages.html" class="external-link">install.packages</a></span><span class="op">(</span><span class="st">"remotes"</span><span class="op">)</span>
<span class="fu">remotes</span><span class="fu">::</span><span class="fu"><a href="https://remotes.r-lib.org/reference/install_github.html" class="external-link">install_github</a></span><span class="op">(</span><span class="st">"labsyspharm/clueR"</span><span class="op">)</span></code></pre></div>
</div>
<div class="section level2">
<h2 id="api-key">API key<a class="anchor" aria-label="anchor" href="#api-key"></a>
</h2>
<p>In order to use this package an API key from Clue is required. For academice purposes, they are freely available at <a href="https://clue.io" class="external-link">Clue</a>.</p>
<p>The API key can be automatically retrieved by all <code>clueR</code> functions if it is added to the <code>~/.Renviron</code> file:</p>
<div class="sourceCode" id="cb2"><pre class="sourceCode R"><code class="sourceCode r"><span id="cb2-1"><a href="#cb2-1" aria-hidden="true" tabindex="-1"></a>CLUE_API_KEY<span class="ot">=</span>xxx</span></code></pre></div>
</div>
<div class="section level2">
<h2 id="example">Example<a class="anchor" aria-label="anchor" href="#example"></a>
</h2>
<p>This is a basic example how to query Clue for a gene signature of interest. Gene signatures can come from any source, but must contain a set of upregulated genes and a set of downregulated genes.</p>
<p>The gene signatures must be converted into <a href="https://software.broadinstitute.org/cancer/software/genepattern/file-formats-guide#GMT" class="external-link">GMT format</a> for use with Clue.</p>
<div class="section level3">
<h3 id="gene-signature-derived-from-differential-expression-with-deseq2">Gene signature derived from differential expression with DESeq2<a class="anchor" aria-label="anchor" href="#gene-signature-derived-from-differential-expression-with-deseq2"></a>
</h3>
<p>In this example, we generate a gene set compatible with Clue from a <a href="https://bioconductor.org/packages/release/bioc/html/DESeq2.html" class="external-link">DESeq2</a> differential expression experiment.</p>
<p>First, we load our example DESeq2 result:</p>
<div class="sourceCode" id="cb3"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va">clueR</span><span class="op">)</span>
<span class="va">deseq2_res_path</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/system.file.html" class="external-link">system.file</a></span><span class="op">(</span><span class="st">"extdata"</span>, <span class="st">"example_deseq2_result.csv.xz"</span>, package <span class="op">=</span> <span class="st">"clueR"</span><span class="op">)</span>
<span class="va">deseq2_res</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/utils/read.table.html" class="external-link">read.csv</a></span><span class="op">(</span><span class="va">deseq2_res_path</span><span class="op">)</span>
<span class="fu"><a href="https://rdrr.io/r/utils/head.html" class="external-link">head</a></span><span class="op">(</span><span class="va">deseq2_res</span><span class="op">)</span>
<span class="co">#&gt;    baseMean log2FoldChange     lfcSE     pvalue      padj gene_id</span>
<span class="co">#&gt; 1 4.3724771    0.738529493 0.6278196 0.01665139 0.1310497       1</span>
<span class="co">#&gt; 2 0.2845104   -0.048028976 0.3494750 0.55657836        NA       2</span>
<span class="co">#&gt; 3 5.4377290   -0.004095729 0.2902083 0.97906352 0.9971952    8086</span>
<span class="co">#&gt; 4 7.9978254    0.498179500 0.3916543 0.03767202 0.2267155   65985</span>
<span class="co">#&gt; 5 3.7455513   -0.167951135 0.3488111 0.29332608 0.6829520   79719</span>
<span class="co">#&gt; 6 6.6533538   -0.264704759 0.3473422 0.18680678 0.5510845   22848</span></code></pre></div>
<p>Note that gene IDs need to be <a href="https://www.ncbi.nlm.nih.gov/gene/" class="external-link">Entrez</a> IDs.</p>
<p>We need to choose a name for the gene set and decide on a <a href="https://bioconductor.org/packages/release/bioc/vignettes/DESeq2/inst/doc/DESeq2.html#p-values-and-adjusted-p-values" class="external-link">significance cutoff (alpha)</a> for our differentially expressed genes.</p>
<div class="sourceCode" id="cb4"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="va">deseq2_gmt</span> <span class="op">&lt;-</span> <span class="fu"><a href="reference/clue_prepare_funs.html">clue_gmt_from_deseq2</a></span><span class="op">(</span><span class="va">deseq2_res</span>, name <span class="op">=</span> <span class="st">"treatment_drug_x"</span>, alpha <span class="op">=</span> <span class="fl">0.05</span><span class="op">)</span>
<span class="co">#&gt; Warning: Coercing gene_id to `character`.</span>
<span class="co">#&gt; Warning: Of 477 genes, 70 are not in BING space</span>
<span class="co">#&gt; Warning: In gene set treatment_drug_x of 477 genes, 70 are not BING genes.</span>
<span class="co">#&gt; Excluding them from analysis. 407 genes left.</span>
<span class="co">#&gt; Warning: In gene set treatment_drug_x, 264 are in the up-regulated list.</span>
<span class="co">#&gt; Maximum is 150. Only keeping the first 150</span>
<span class="fu"><a href="https://rdrr.io/r/utils/str.html" class="external-link">str</a></span><span class="op">(</span><span class="va">deseq2_gmt</span><span class="op">)</span>
<span class="co">#&gt;  Named chr [1:2] "/var/folders/_h/wpz1qzm12t5687lbdm87lsh00000gp/T//RtmpbwGbaz/file65a41ca924d.gmt" ...</span>
<span class="co">#&gt;  - attr(*, "names")= chr [1:2] "down" "up"</span></code></pre></div>
<p>A number of warnings are raised, indicating that some gene IDs are not part of the <a href="https://clue.io/connectopedia/category/Concept" class="external-link">BING gene space</a>. These genes are removed from the gene sets and not considered by Clue.</p>
<p><code>clue_prepare_deseq2</code> returns a named vector with paths to the GMT files for the up- and the down-regulated genes.</p>
</div>
<div class="section level3">
<h3 id="gene-signature-derived-from-pre-existing-lists">Gene signature derived from pre-existing lists<a class="anchor" aria-label="anchor" href="#gene-signature-derived-from-pre-existing-lists"></a>
</h3>
<p>We can also convert an pre-existing set of genes from any source to GMT files:</p>
<div class="sourceCode" id="cb5"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="va">up_genes</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span>
  <span class="st">"10365"</span>, <span class="st">"1831"</span>, <span class="st">"9314"</span>, <span class="st">"4846"</span>, <span class="st">"678"</span>, <span class="st">"22992"</span>, <span class="st">"3397"</span>, <span class="st">"26136"</span>, 
  <span class="st">"79637"</span>, <span class="st">"5551"</span>, <span class="st">"7056"</span>, <span class="st">"79888"</span>, <span class="st">"1032"</span>, <span class="st">"51278"</span>, <span class="st">"64866"</span>, <span class="st">"29775"</span>, 
  <span class="st">"994"</span>, <span class="st">"51696"</span>, <span class="st">"81839"</span>, <span class="st">"23580"</span>, <span class="st">"219654"</span>, <span class="st">"57178"</span>, <span class="st">"7014"</span>, 
  <span class="st">"57513"</span>, <span class="st">"51599"</span>, <span class="st">"55818"</span>, <span class="st">"4005"</span>, <span class="st">"4130"</span>, <span class="st">"4851"</span>, <span class="st">"2050"</span>, <span class="st">"50650"</span>, 
  <span class="st">"9469"</span>, <span class="st">"54438"</span>, <span class="st">"3628"</span>, <span class="st">"54922"</span>, <span class="st">"3691"</span>, <span class="st">"65981"</span>, <span class="st">"54820"</span>, <span class="st">"2261"</span>, 
  <span class="st">"2591"</span>, <span class="st">"7133"</span>, <span class="st">"162427"</span>, <span class="st">"10912"</span>, <span class="st">"8581"</span>, <span class="st">"2523"</span>, <span class="st">"25807"</span>, <span class="st">"9922"</span>, 
  <span class="st">"30850"</span>, <span class="st">"4862"</span>, <span class="st">"8567"</span>, <span class="st">"79686"</span>, <span class="st">"55615"</span>, <span class="st">"51283"</span>, <span class="st">"3337"</span>, <span class="st">"2887"</span>, 
  <span class="st">"3223"</span>, <span class="st">"6915"</span>, <span class="st">"6907"</span>, <span class="st">"26056"</span>, <span class="st">"259217"</span>, <span class="st">"6574"</span>, <span class="st">"23097"</span>, <span class="st">"5164"</span>, 
  <span class="st">"57493"</span>, <span class="st">"7071"</span>, <span class="st">"5450"</span>, <span class="st">"113146"</span>, <span class="st">"8650"</span>
<span class="op">)</span>
<span class="va">down_genes</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span>
  <span class="st">"5128"</span>, <span class="st">"5046"</span>, <span class="st">"956"</span>, <span class="st">"10426"</span>, <span class="st">"9188"</span>, <span class="st">"23403"</span>, <span class="st">"7204"</span>, <span class="st">"1827"</span>, 
  <span class="st">"3491"</span>, <span class="st">"9076"</span>, <span class="st">"330"</span>, <span class="st">"8540"</span>, <span class="st">"22800"</span>, <span class="st">"10687"</span>, <span class="st">"19"</span>, <span class="st">"63875"</span>, 
  <span class="st">"10979"</span>, <span class="st">"51154"</span>, <span class="st">"10370"</span>, <span class="st">"50628"</span>, <span class="st">"7128"</span>, <span class="st">"6617"</span>, <span class="st">"7187"</span>, <span class="st">"22916"</span>, 
  <span class="st">"81034"</span>, <span class="st">"58516"</span>, <span class="st">"3096"</span>, <span class="st">"4794"</span>, <span class="st">"5202"</span>, <span class="st">"26511"</span>, <span class="st">"8767"</span>, <span class="st">"2355"</span>, 
  <span class="st">"22943"</span>, <span class="st">"1490"</span>, <span class="st">"133"</span>, <span class="st">"11010"</span>, <span class="st">"51025"</span>, <span class="st">"23160"</span>, <span class="st">"56902"</span>, <span class="st">"3981"</span>, 
  <span class="st">"5209"</span>, <span class="st">"6347"</span>, <span class="st">"5806"</span>, <span class="st">"7357"</span>, <span class="st">"9425"</span>, <span class="st">"3399"</span>, <span class="st">"6446"</span>, <span class="st">"64328"</span>, 
  <span class="st">"6722"</span>, <span class="st">"8545"</span>, <span class="st">"688"</span>, <span class="st">"861"</span>, <span class="st">"390"</span>, <span class="st">"23034"</span>, <span class="st">"51330"</span>, <span class="st">"51474"</span>, 
  <span class="st">"2633"</span>, <span class="st">"4609"</span>
<span class="op">)</span>

<span class="va">pre_gmt</span> <span class="op">&lt;-</span> <span class="fu"><a href="reference/clue_prepare_funs.html">clue_gmt_from_list</a></span><span class="op">(</span><span class="va">up_genes</span>, <span class="va">down_genes</span>, <span class="st">"my_favourite_genes"</span><span class="op">)</span>
<span class="fu"><a href="https://rdrr.io/r/utils/str.html" class="external-link">str</a></span><span class="op">(</span><span class="va">pre_gmt</span><span class="op">)</span>
<span class="co">#&gt;  Named chr [1:2] "/var/folders/_h/wpz1qzm12t5687lbdm87lsh00000gp/T//RtmpbwGbaz/file65a5c31574.gmt" ...</span>
<span class="co">#&gt;  - attr(*, "names")= chr [1:2] "down" "up"</span></code></pre></div>
</div>
<div class="section level3">
<h3 id="querying-clue">Querying Clue<a class="anchor" aria-label="anchor" href="#querying-clue"></a>
</h3>
<p>Now that we have GMT files, we can query Clue. Here we use the GMT files derived from the DESeq2 result, but we could also use the <code>pre_gmt</code> files generated above.</p>
<div class="sourceCode" id="cb6"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="va">submission_result</span> <span class="op">&lt;-</span> <span class="fu"><a href="reference/clue_query_submit.html">clue_query_submit</a></span><span class="op">(</span>
  <span class="va">deseq2_gmt</span><span class="op">[[</span><span class="st">"up"</span><span class="op">]</span><span class="op">]</span>, <span class="va">deseq2_gmt</span><span class="op">[[</span><span class="st">"down"</span><span class="op">]</span><span class="op">]</span>, name <span class="op">=</span> <span class="st">"deseq2_query_job"</span>
<span class="op">)</span>
<span class="va">submission_result</span><span class="op">$</span><span class="va">result</span><span class="op">$</span><span class="va">job_id</span>
<span class="co">#&gt; [1] "5d36096d12e15519133e87fd"</span></code></pre></div>
<p><code>clue_query_submit</code> returns a nested list containing information about the submitted job, including the job id.</p>
<p>Now we have to wait for the job to finish. We can use the function <code>clue_query_wait</code> to pause our script until the results are ready.</p>
<div class="sourceCode" id="cb7"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="fu"><a href="reference/clue_query_poll.html">clue_query_wait</a></span><span class="op">(</span><span class="va">submission_result</span>, interval <span class="op">=</span> <span class="fl">60</span>, timeout <span class="op">=</span> <span class="fl">600</span><span class="op">)</span>
<span class="co">#&gt; Job not completed yet, waiting for: 5d36096d12e15519133e87fd</span>
<span class="co">#&gt; Job not completed yet, waiting for: 5d36096d12e15519133e87fd</span>
<span class="co">#&gt; Job not completed yet, waiting for: 5d36096d12e15519133e87fd</span>
<span class="co">#&gt; Job completed: 5d36096d12e15519133e87fd</span></code></pre></div>
<p><code>clue_query_wait</code> will automatically continue execution of the script once results are ready. Now we can download and parse the results:</p>
<div class="sourceCode" id="cb8"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="va">result_path</span> <span class="op">&lt;-</span> <span class="fu"><a href="reference/clue_query_download.html">clue_query_download</a></span><span class="op">(</span><span class="va">submission_result</span><span class="op">)</span>
<span class="va">result_df</span> <span class="op">&lt;-</span> <span class="fu"><a href="reference/clue_parse_result.html">clue_parse_result</a></span><span class="op">(</span><span class="va">result_path</span><span class="op">)</span>
<span class="co">#&gt; reading /var/folders/_h/wpz1qzm12t5687lbdm87lsh00000gp/T//RtmpbwGbaz/clueR-65a4facb13e/my_analysis.sig_fastgutc_tool.5d36096d12e15519133e87fd/matrices/gutc/ns_pert_summary.gctx</span>
<span class="co">#&gt; done</span></code></pre></div>
<table class="table">
<thead><tr class="header">
<th align="left">pert_id</th>
<th align="left">pert_type</th>
<th align="left">pert_iname</th>
<th align="left">gene_set</th>
<th align="right">ns</th>
</tr></thead>
<tbody>
<tr class="odd">
<td align="left">BRD-A09094913</td>
<td align="left">trt_cp</td>
<td align="left">strychnine</td>
<td align="left">treatment_drug_x</td>
<td align="right">0.0000000</td>
</tr>
<tr class="even">
<td align="left">BRD-A55393291</td>
<td align="left">trt_cp</td>
<td align="left">testosterone</td>
<td align="left">treatment_drug_x</td>
<td align="right">0.5971081</td>
</tr>
<tr class="odd">
<td align="left">BRD-A93255169</td>
<td align="left">trt_cp</td>
<td align="left">thalidomide</td>
<td align="left">treatment_drug_x</td>
<td align="right">0.1799593</td>
</tr>
<tr class="even">
<td align="left">BRD-K41731458</td>
<td align="left">trt_cp</td>
<td align="left">triclosan</td>
<td align="left">treatment_drug_x</td>
<td align="right">0.1040271</td>
</tr>
<tr class="odd">
<td align="left">BRD-K63675182</td>
<td align="left">trt_cp</td>
<td align="left">triflupromazine</td>
<td align="left">treatment_drug_x</td>
<td align="right">-0.7581666</td>
</tr>
<tr class="even">
<td align="left">BRD-A19195498</td>
<td align="left">trt_cp</td>
<td align="left">trimipramine</td>
<td align="left">treatment_drug_x</td>
<td align="right">-0.5755145</td>
</tr>
<tr class="odd">
<td align="left">BRD-K99621550</td>
<td align="left">trt_cp</td>
<td align="left">tubocurarine</td>
<td align="left">treatment_drug_x</td>
<td align="right">0.0000000</td>
</tr>
<tr class="even">
<td align="left">BRD-A51714012</td>
<td align="left">trt_cp</td>
<td align="left">venlafaxine</td>
<td align="left">treatment_drug_x</td>
<td align="right">0.3056950</td>
</tr>
<tr class="odd">
<td align="left">BRD-K45117373</td>
<td align="left">trt_cp</td>
<td align="left">Y-26763</td>
<td align="left">treatment_drug_x</td>
<td align="right">0.0000000</td>
</tr>
<tr class="even">
<td align="left">BRD-K42095107</td>
<td align="left">trt_cp</td>
<td align="left">daidzein</td>
<td align="left">treatment_drug_x</td>
<td align="right">0.0000000</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="section level2">
<h2 id="funding">Funding<a class="anchor" aria-label="anchor" href="#funding"></a>
</h2>
<p>We gratefully acknowledge support by NIH Grant 1U54CA225088-01: Systems Pharmacology of Therapeutic and Adverse Responses to Immune Checkpoint and Small Molecule Drugs.</p>
</div>
</div>
  </main><aside class="col-md-3"><div class="license">
<h2 data-toc-skip>License</h2>
<ul class="list-unstyled">
<li><a href="LICENSE.html">Full license</a></li>
<li><small><a href="https://opensource.org/licenses/mit-license.php" class="external-link">MIT</a> + file <a href="LICENSE-text.html">LICENSE</a></small></li>
</ul>
</div>


<div class="citation">
<h2 data-toc-skip>Citation</h2>
<ul class="list-unstyled">
<li><a href="authors.html#citation">Citing cluequery</a></li>
</ul>
</div>

<div class="developers">
<h2 data-toc-skip>Developers</h2>
<ul class="list-unstyled">
<li>Clemens Hug <br><small class="roles"> Author, maintainer </small> <a href="https://orcid.org/0000-0002-8299-3274" target="orcid.widget" aria-label="ORCID" class="external-link"><span class="fab fa-orcid orcid" aria-hidden="true"></span></a> </li>
</ul>
</div>



  </aside>
</div>


    <footer><div class="pkgdown-footer-left">
  <p></p>
<p>Developed by Clemens Hug.</p>
</div>

<div class="pkgdown-footer-right">
  <p></p>
<p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.0.1.</p>
</div>

    </footer>
</div>

  

  

  </body>
</html>
